cpp/src/io/comp/gpuinflate.h

```diff
@@ -16,26 +16,26 @@
 
 #pragma once
 
-#include <cstdint>
+#include <cudf/utilities/span.hpp>
 
 #include <rmm/cuda_stream_view.hpp>
 
+#include <cstdint>
+
 namespace cudf {
 namespace io {
 /**
  * @brief Input parameters for the decompression interface
  */
-struct gpu_inflate_input_s {
-  const void* srcDevice;
-  uint64_t srcSize;
-  void* dstDevice;
-  uint64_t dstSize;
+struct device_decompress_input {
```

Comment:
Been thinking about this, and I'm still not happy with the name of this struct:

1. Turns out it's also used for compression. 
2. It's just a pair of device buffer views, there's nothing specific to (de)compression.

Any suggestions?

Reply:
That makes sense. I was thinking of merging dst and status as output, but having all of them separate is even better 👍